### ISSUE TITLE
v0.0.5 configure for jsonapi_indifferent_hash breaking rake spec

### ISSUE THREAD

**Issue by sa73917** [jsonapi-rb/jsonapi-rspec#20]
## Expected Behavior

jsonapi-rspec gem should load within rake without issues

## Actual Behavior

```rake spec
rake aborted!
NoMethodError: undefined method `configure' for RSpec:Module
.../config/application.rb:20:in `<top (required)>'
.../Rakefile:4:in `require_relative'
.../Rakefile:4:in `<top (required)>'

Caused by:
LoadError: cannot load such file -- jsonapi-rspec
.../config/application.rb:20:in `<top (required)>'
.../Rakefile:4:in `require_relative'
.../Rakefile:4:in `<top (required)>'
(See full trace by running task with --trace)
```

## Steps to Reproduce the Problem

  1. include v0.0.5 in project Gemfile
  2. run rake spec

## Specifications

  - Version: v0.0.5
  - Ruby version: 2.7.1
  - Rake version: 13.0.1

## Possible solution

[requiring rspec/core in rspec.rb](https://github.com/sa73917/jsonapi-rspec/commit/606f5ae3428c3a81366f42b9a63e949d8ea43791) resolves the issue.  Not sure if this is the best solution but certainly a simple fix.

**Comment by stas** [comment]
Hmm, could it be because of this:
https://github.com/jsonapi-rb/jsonapi-rspec/blob/master/lib/jsonapi/rspec.rb#L11-L13

Could you please open a PR please.

**Comment by sa73917** [comment]
Yes its definitely those lines of code - it's just rake is being a bit eager trying to load the gem before the RSpec:Core module is there...  

Not sure if there's a better way but [this PR resolves the issue](https://github.com/jsonapi-rb/jsonapi-rspec/pull/21)

**Comment by stas** [comment]
Fixed in #21

**Comment by stas** [comment]
Thank you @sa73917!
